manifest.yaml

```diff
@@ -343,6 +343,11 @@ packages:
  # https://bugzilla.redhat.com/show_bug.cgi?id=1925698
  # https://github.com/openshift/machine-config-operator/pull/2421
  - conntrack-tools
+ # https://github.com/openshift/os/issues/822
+ # https://bugzilla.redhat.com/show_bug.cgi?id=1972102
+ # Upstream PR https://github.com/coreos/fedora-coreos-config/pull/786
```

Comment:
Let's keep only this reference. All the others are fine in the commit message only.